1-exercises/A-undefined/exercise.js

```diff
@@ -12,25 +12,27 @@
 // Example 1
 let a;
 console.log(a);
-
+//variable a need to be assigned a value
 
 // Example 2
 function sayHello() {
     let message = "Hello";
 }
+//sayHello doesnt return anything. variable message is just in the function's scope but, it is not consolled or returned
 
 let hello = sayHello();
 console.log(hello);
-
+// the function sayHello doesnt return any value
 
 // Example 3
 function sayHelloToUser(user) {
     console.log(`Hello ${user}`);
 }
-
 sayHelloToUser();
+// user variable is not assigned, doesnt have a value
 
```

Comment:
It is better to say that when we call the function, there is no any arguments - there's nothing in the brackets! Therefore value of the user parameter inside the function is undefined